README.md

```diff
@@ -46,3 +47,15 @@ To Update the `kubernetes-mixin` dashboards:
 
 * Follow the instructions in [giantswarm-kubernetes-mixin](https://github.com/giantswarm/giantswarm-kubernetes-mixin)
 * Run `./scripts/sync-mixins.sh (?my-fancy-branch-or-tag)` to updated the `helm/dashboards/dashboards/mixin` folder.
+
+## Origins of the dashboards
+
+Some dashboards are crafted by us (Giantswarm) or forked from public dashboards.
+
+Some dashboards come from mixins, with a few manual updates. They have a `.*-mixin` tag, like `kubernetes-mixin` or `prometheus-mixin`.
+
+Beware when updating them: even though they come from mixins, they may require tweaks to work with our architecture.
+
+The dashboards that don't require tweaks can be automatically updated from pre-generated mixins at https://monitoring.mixins.dev/ with the `update-mixins-monitoring` github workflow.
+
+In the future we should provide documentation and automation to perform clean updates of mixins-based dashboards.
```

Comment:
As you mentioned before we seemed to have automation for updating mixins. Or I don't get what we are talking about here?

Reply:
Well... for some, we have scripts, that may work. Or not. Depends if people have manually updated the dashboards in the meantime.
So my idea of the procedure is:
- `git log` or `git blame` to check what/when the dashboards where changed.
- run the script
- look at diff
- cross fingers

But according to the readme, the script `./scripts/sync-mixins.sh` is for grafana cloud dashboards, which means you can't even `git blame`. I think there's a changelog though.

There's too much unknown there, I prefer not to dive in for the moment, that's why I didn't get into details.

Reply:
I like the crossing finger part 😁

Ok this is more like warning that the automation is not perfect.